simple_network_sim/network_of_populations.py

```diff
@@ -95,6 +128,16 @@ def getAges(node):
 
 # CurrentlyInUse
 def getTotalInAge(nodeState, ageTest):
+    """Get the size of the population within an age range.
```

Comment:
I know the class is called "age range" but I think saying "age group" in the documentation would be clearer.